apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/Rates/index.tsx

```diff
@@ -60,7 +62,11 @@ export default function ProviderRate({
   const [defaultPartner, setDefaultPartner] = useState<string | null>(null);
   const selectedRate = useSelector(rateSelector);
   const filteredRates = useMemo(() => filterRates(rates, filter), [rates, filter]);
-
+  const providers = rates?.map(rate => rate.provider) ?? [];
```

Comment:
2 rates can have the same provider (float and fix)

Reply:
Thank you, will get unique providers here 🙌

Reply:
Sorry, forgot to push these changes yesterday. Updated now.